README.md

```diff
@@ -275,6 +276,57 @@ function loadDataAsProps({ store: { dispatch } }) {
 export default connectAsync({ loadDataAsProps })(Articles);
 ```
 
+### Method Opts
+
+Some REST APIs support bulk updates as opposed to individual resource updates. There are many thoughts on this matter but they generally suggest using a POST or a PUT for this type of CRUD action. For iguazu-rest, we take the opinion that by default, updating a collection should be a POST but provide the option to override the method. However, other CRUD actions' methods cannot be overriden (e.g. `loadCollection`).
```

Comment:
What is the difference between updateCollection and createResource from an API standpoint if both are using POST and a URL without an ID? Just different request bodies, object vs array?

Reply:
From an API standpoint, that would be correct. The API would see the single object vs an array.